app/code/PalashAs/AskQuestion/view/frontend/web/js/ask-question.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'jquery',
+    'jquery/ui',
+    'Magento_Ui/js/modal/alert',
+    'mage/cookies',
+    'mage/translate'
+], function ($) {
+    'use strict';
+
+    $.widget('palash.askquestion_widget', $.mage.alert, {
+        options: {
+            cookieName: 'ask_question_timestamp'
+        },
+
+        /** @inheritdoc */
+        _create: function () {
+            $(this.element).submit(this.submitForm.bind(this));
+        },
+
+        /**
+         * Validate request and submit the form if able
+         */
+        submitForm: function () {
+            if (!this.validateForm()) {
+                return;
+            }
+           
+            this.ajaxSubmit();
+        },
+
+        /**
+         * Validate request form
+         */
+        validateForm: function () {
+            return $(this.element).validation().valid();
+        },
+
+        /**
+         * Submit request via AJAX. Add form key to the post data.
+         */
+        ajaxSubmit: function () {
+            var formData = new FormData($(this.element).get(0));
+
+            formData.append('form_key', $.mage.cookies.get('form_key'));
+            formData.append('ask_question_timestamp', $.mage.cookies.get(this.options.cookieName));
```

Comment:
It is much better to validate timestamp here and not to send AJAX request on the server at all if 120 seconds have not passed yet. Besides, you will not get cookie value at all if it is expired. So, all you need is to set cookie lifetime to 120 seconds and that's it!